axoned/scripts/helpers.sh

```
@@ -0,0 +1,138 @@
+# File: $WORK_DIR_AXONE/scripts/helpers.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)